ledger/committee/src/lib.rs

```diff
@@ -50,7 +50,7 @@ pub const MAX_DELEGATORS: u32 = 100_000u32;
 pub struct Committee<N: Network> {
     /// The committee ID, defined as the hash of the starting round, members, and total stake.
     id: Field<N>,
-    /// The starting round number for this committee.
+    /// The round number (of the block) at which this committee is created (always even).
```

Comment:
nit: shouldn't it be "was created"? If the committee struct is instantiated, I assume the committee exists?